packages/snaps-simulation/package.json

```diff
@@ -0,0 +1,100 @@
+{
+  "name": "@metamask/snaps-simulation",
+  "version": "0.0.0",
+  "repository": {
```

Comment:
It might be good to add description to this package.

Reply:
Done!